tests/test_stormwater.py

```diff
@@ -798,6 +798,34 @@ def test_validate(self):
             if arg_list[0] in ['retention_radius', 'road_centerlines_path']:
                 self.assertEqual(message, validation.MESSAGES['MISSING_VALUE'])
 
+    def test_validate_noninteger_soil_raster(self):
+        """Stormwater: test arg validation."""
+        from natcap.invest import stormwater, validation
+
+        soil_array = numpy.array([[1, 2], [3, 4]], dtype=numpy.float32)
+        soil_path = os.path.join(self.workspace_dir, 'soils.tif')
+        to_raster(soil_array, soil_path, pixel_size=(10, -10))
+
+        # test args missing necessary values for adjust ratios
```

Comment:
This comment seems out of place. I wouldn't want us to be confused in the future if this is not actually what's being tested here.

Reply:
Good catch, I accidentally pasted that in from a different test.